Configuration/Eras/python/Era_Run2_2017_cff.py

```diff
@@ -11,10 +11,11 @@
 from Configuration.Eras.Modifier_run2_GEM_2017_cff import run2_GEM_2017
 from Configuration.Eras.Modifier_stage2L1Trigger_2017_cff import stage2L1Trigger_2017
 from Configuration.Eras.Modifier_run2_HLTconditions_2017_cff import run2_HLTconditions_2017
+from Configuration.Eras.Modifier_run2_HLTconditions_2016_cff import run2_HLTconditions_2016
 from Configuration.Eras.Modifier_run2_muon_2017_cff import run2_muon_2017
 from Configuration.Eras.Modifier_run2_muon_2016_cff import run2_muon_2016
 
-Run2_2017 = cms.ModifierChain(Run2_2016.copyAndExclude([run2_muon_2016]), 
+Run2_2017 = cms.ModifierChain(Run2_2016.copyAndExclude([run2_muon_2016, run2_HLTconditions_2016]), 
```

Comment:
@skonstantinou could you please clarify the purpose of this modification? Add a piece to drop it.

Reply:
The run2_HLTcondition_2016 was contained in Run2_2018 and Run2_2017 and thus it was always selected, even when using the Run2_2017 or Run2_2018 era option. The problem is solved by removing the run2_HLTconditions_2016 modifier from Run2_2017 and run2_HLTconditions_2017 from Run2_2018 with this modification (similarly to what was done in #26489).